java/pom.xml

```diff
@@ -48,13 +48,18 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.13.4</version>
+            <version>2.13.4.1</version>
```

Comment:
Weird, there are 4 bits in this version?

Reply:
Yeah it's weird. That's even what the bot says to use: https://github.com/anvilco/anvil-api-usage-examples/security/dependabot/3

Guess they're not on semver.